ui/app/src/components/Navigation/GlobalNav.tsx

```diff
@@ -506,7 +506,7 @@ export default function GlobalNav(props: GlobalNavProps) {
                   sites.map((site, i) => (
                     <SiteCard
                       key={i}
-                      title={site.name}
+                      title={site.name??site.id}
```

Comment:
On next code, this always comes from the new JS services so it's unnecessary on UI next pieces.